src/models/room.ts

```diff
@@ -127,6 +128,7 @@ export enum RoomEvent {
     OldStateUpdated = "Room.OldStateUpdated",
     CurrentStateUpdated = "Room.CurrentStateUpdated",
     HistoryImportedWithinTimeline = "Room.historyImportedWithinTimeline",
+    UnreadNotifications = "Room.UnreadNotifications",
```

Comment:
it bothers me that we mix `Subject.lowerCamelCase` and `Subject.UpperCamelCase`, ftr. Personally I prefer `lowerCamelCase`, but that seems to be outvoted here.